make/autoconf/jdk-options.m4

```diff
@@ -228,23 +228,6 @@ AC_DEFUN_ONCE([JDKOPT_SETUP_JDK_OPTIONS],
 ])
 
 ###############################################################################
-#
-# Enable or disable the elliptic curve crypto implementation
-#
-AC_DEFUN_ONCE([JDKOPT_DETECT_INTREE_EC],
```

Comment:
There should be a call to this macro from either configure.ac or this file that also needs to be removed.

Reply:
found it in configure.ac and removed